manager/ui/hawtio/plugins/api-manager/ts/forms/edit-gateway.ts

```diff
@@ -89,6 +89,7 @@ module Apiman {
                 $scope.updateButton.state = 'in-progress';
                 var gateway = Gateway();
                 ApimanSvcs.update({ entityType: 'gateways', secondaryType: $scope.gateway.id }, gateway, function() {
+                    $rootScope.isDirty = false;
```

Comment:
@EricWittmann - Thanks for tagging me on this. That was a good catch, I hope we don't have that issue in other areas. I'll do some additional testing later today. :)